web-components/src/components/table/Table.ts

```diff
@@ -31,7 +31,7 @@ export class Table extends LitElement {
   results: any;
   config = {
     quoteChar: '"',
-    escapeChar: '"',
+    escapeChar: '""',
```

Comment:
@helen-v sorry if this is just a bad assumption, but my guess would be that an 'escapeChar' would be the slash  `\` rather than the quote? Is this a specific table thing that I'm just not privy to maybe?

Reply:
For parse scv I use Papaprse library, and set this configuration according documentation: https://www.papaparse.com/docs#config